src/server/serverElectron.ts

```diff
@@ -0,0 +1,35 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import express = require('express');
+import bodyParser = require('body-parser');
+import cors = require('cors');
+import { dataPlaneUri,
+    handleDataPlanePostRequest,
+    cloudToDeviceUri,
+    handleCloudToDevicePostRequest,
+    eventHubMonitorUri,
+    handleEventHubMonitorPostRequest,
+    eventHubStopUri,
+    handleEventHubStopPostRequest,
+    modelRepoUri,
+    handleModelRepoPostRequest
+} from './serverHelper';
+
+const SERVER_PORT = 8081;
+const app = express();
+
+app.use(bodyParser.json());
```

Comment:
Please rename serverHelper.ts to serverBase.ts. Also, you can move all this logic into an init function that takes serverPort as an argument. Then serverLocal and serverElectron get refactored to:

```
import { init } from './serverBase';
const SERVER_PORT = 808x;
init(SERVER_PORT);
```

Reply:
makes total sense.

Reply:
Made a class and assign server_port in the constructor here :)